.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "_docs"]
+	path = _docs
+	url = https://github.com/scorbett123/docs.git
```

Comment:
This would need to be changed to a custom docs repo, however I thought that a submodule would mean that the docs could be separated out meaning that they are easier to contribute to.